patch/0067-platform-mellanox-Add-dedicated-match-for-system-typ.patch

```diff
@@ -0,0 +1,434 @@
+From 8e3d1dbfd9cf686f9282d9b36f4ccc5ef3949ef5 Mon Sep 17 00:00:00 2001
+From: Vadim Pasternak <vadimp@nvidia.com>
+Date: Sun, 7 Feb 2021 22:59:53 +0200
+Subject: [PATCH backport 5.10 067/182] platform/mellanox: Add dedicated match
+ for system type QMB8700 (downstream)
+
+Use dedicated match function for QMB8700 system in order to work-around
+wrong CPLD register map.
+
+Note this is downstream kernel patch, no intention to upstream this
+code.
```

Comment:
Can you please elaborate why?

Reply:
System QMB8700 is not used by Sonic and not used by our up-stream customers.
It has some hardware bug in CPLD and this was the reason to add dedicated patch to WA this bug.

The problem that after this patch many other patches have been added to mlx-platform driver and removing this patch will create conflicts of applying later patches.
So, we keep this patch to avoid such conflicts.

In kernel v6.1 we’ll keep this patch on top of all other patches to avoid such sort of conflicts.
But in v5.10 it is very hard now to rebase all the patches for moving this one to the top.

Reply:
If you could add that to the commit message body of the patch, that would be even better.

Thank you for improving the situation for Linux 6.1.

Reply:
Sure, will add.
Thanks.